src/db/spanner/spanner.rs

```diff
@@ -0,0 +1,74 @@
+extern crate google_spanner1 as spanner1;
```

Comment:
A couple more nits: 
- we should be able to kill all the extern declarations here
- let's rename this module now (maybe r2d2 or manager.rs?)
- maybe some import cleanups (import ManageConnection and the hyper bits directly instead of using their full namespaces inline and kill the unneeded std::result from std::result::Result)